ITEventsTests/FavoriteEventsPresenterTests/FavoriteEventsPresenterTests.swift

```diff
@@ -0,0 +1,107 @@
+import XCTest
+@testable import Promises
+@testable import ITEvents
+
+class FavoriteEventsPresenterTests: XCTestCase {
+    private var presenter: IFavoritePresenter!
+    private var viewMock: FavoriteViewMock!
+    private var selectedEventServiceMock: SelectedEventServiceMock!
+    private var eventStorageMock: EventStorageMock!
+    
+    override func setUp() {
+        super.setUp()
+        
+        viewMock = FavoriteViewMock()
+        selectedEventServiceMock = SelectedEventServiceMock()
+        eventStorageMock = EventStorageMock()
```

Comment:
it is better to have event list with test data stored locally in each test class, instead of stored globally in EventStorageMock class:
```swift
let events = createTestEvents()
eventStorageMock = EventStorageMock(events: events)
```